modules/ROOT/pages/planning-and-tuning/query-tuning/indexes.adoc

```diff
@@ -187,6 +187,10 @@ However, other predicates are only used when it is known that the property is co
 
 This means that a `TEXT` index is not able to solve e.g. `a.prop = b.prop`, unless a type constraint also exists on the property.
 
+One can add xref:values-and-types/type-predicate.adoc[type predicate expressions] such as `IS {two-colons} STRING NOT NULL` to enforce the String type and possibly the property's existence.
+Note that `null IS {two-colons} STRING` and that indexes do not store entities with `null` properties.
```

Comment:
Is there something missing in this sentence or is it me? I don't understand it

Reply:
I guess that is Mathematician slang: `null IS :: STRING` is a statement. One could add a "holds" or "is true" here, to make it less geeky:
```suggestion
Note that `null IS {two-colons} STRING` holds and that indexes do not store entities with `null` properties.
```

Reply:
Ah right, yes, I see it now! I did not read `IS` as _is_, I think it's more readable if you add "holds" or "is true" at the end.

Reply:
Done in 31c03ee.